package.json

```
@@ -22,14 +22,13 @@
   "author": "pcaversaccio (https://pcaversaccio.com), Matt Solomon (https://mattsolomon.dev)",
   "license": "AGPL-3.0-only",
   "scripts": {
-    "help": "npx hardhat help",
-    "accounts": "npx hardhat accounts",
-    "balances": "npx hardhat balances",
-    "node": "npx hardhat node",
-    "clean": "npx hardhat clean",
+    "clean:hh": "npx hardhat clean",
+    "clean:forge": "forge clean",
```

What are the use cases for two separate commands, as opposed to just
```suggestion
    "clean": "npx hardhat clean && forge clean",
```

---

implemented  via https://github.com/pcaversaccio/createx/pull/18/commits/b8c3646a295418d978ea321ceee83aeb72a907fb